test/integration/targets/postgresql/tasks/postgresql_facts.yml

```diff
@@ -0,0 +1,101 @@
+# Test code for the postgresql_facts module
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: "postgresql_facts - test return values"
```

Comment:
So not something you have to change, but I noticed this playbook is quoting a lot of things that do not require quoting at all. For documentation and examples we recommend to only quote where it is required by YAML to quote.
This helps users to understand when quotes are needed.